zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/identity/PermissionsBehavior.java

```diff
@@ -14,24 +14,16 @@
 import io.camunda.zeebe.engine.state.immutable.ProcessingState;
 import io.camunda.zeebe.protocol.impl.record.value.authorization.AuthorizationRecord;
 import io.camunda.zeebe.protocol.record.RejectionType;
-import io.camunda.zeebe.protocol.record.value.AuthorizationRecordValue.PermissionValue;
 import io.camunda.zeebe.protocol.record.value.AuthorizationResourceType;
 import io.camunda.zeebe.protocol.record.value.PermissionType;
 import io.camunda.zeebe.stream.api.records.TypedRecord;
 import io.camunda.zeebe.util.Either;
-import java.util.HashSet;
 import java.util.Set;
 
 public class PermissionsBehavior {
 
-  public static final String OWNER_NOT_FOUND_MESSAGE =
-      "Expected to find owner with key: '%d', but none was found";
-  public static final String PERMISSION_ALREADY_EXISTS_MESSAGE =
-      "Expected to add '%s' permission for resource '%s' and resource identifiers '%s' for owner '%s', but this permission for resource identifiers '%s' already exist. Existing resource ids are: '%s'";
-  public static final String PERMISSION_NOT_FOUND_MESSAGE =
-      "Expected to remove '%s' permission for resource '%s' and resource identifiers '%s' for owner '%s', but this permission for resource identifiers '%s' is not found. Existing resource ids are: '%s'";
-  public static final String AUTHORIZATION_ALREADY_EXISTS_MESSAGE =
-      "Expected to create authorization for owner '%s' with permission type '%s' and resource type '%s', but this permission for resource identifiers '%s' already exist. Existing resource ids are: '%s'";
+  public static final String PERMISSIONS_ALREADY_EXISTS_MESSAGE =
+      "Expected to created authorization for owner '%s' for resource identifier '%s', but an authorization for this resource identifier already exists.";
```

Comment:
```suggestion
      "Expected to create authorization for owner '%s' for resource identifier '%s', but an authorization for this resource identifier already exists.";
```

🤡 Small typo

Reply:
This'll also fix your failing unit test 😄

Reply:
🤦